module-2/backend/week_four.md

```diff
@@ -10,14 +10,64 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+
+  A cookie is a piece of data assigned to and saved in your browser when you visit a website
+
 * What’s the difference between a session and a cookie?
+
+  a session is encrypted, and can hold more data(?)
```

Comment:
Mostly the encrypted part. Intrigued by more data? I haven't heard that, but could be something I just haven't heard yet.